src/pages/hardware/developingAnIOX/messagingProtocol.tsx

```diff
@@ -0,0 +1,1361 @@
+        <h4>Payload</h4>
+        <table>
+            <thead>
+                <tr>
+                    <th>Byte #</th>
+                    <th>Byte Description</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>0 - Bit 0</td>
+                    <td>
+                        0 = Clear to send more RX Data.
+                        <br></br>
+                        1 = Stop sending UART Data. Buffer 80% full, withhold next frame 50 ms. 
+                    </td>
+                </tr>
+                <tr>
+                    <td>1 - Bit 1-7</td>
+                    <td>Reserved</td>
+                </tr>
+            </tbody>
+        </table>
+        <h3>Application Specific Data (0x1C)</h3>
+        <p>Sent by the GO device after a packet wrapped passthrough message attempt to the server. A 'rejected' response from the modem typically means it is not connected. If the message is 'accepted', this means it was added to the modem's TCP socket buffer. It is NOT a confirmation that the message was successfully sent.</p>
+        <h4>Type 0: Modem Transmission Result</h4>
+        <table>
+            <thead>
+                <tr>
+                    <th>
+                        <strong>Byte #</strong>
+                    </th>
+                    <th>
+                        <strong>Byte Description</strong>
+                    </th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>0</td>
+                    <td>Log Type</td>
+                </tr>
+                <tr>
+                    <td>1</td>
+                    <td>
+                        0 = Rejected
+                        <br></br>
+                        1 = Accepted
+                    </td>
+                </tr>
+            </tbody>
+        </table>
+        <h3>IOX Single Frame Log Data (0x1D)</h3>
+        <p>Sent from the IOX to the GO device when the IOX wants to create a log that can fit into a single CAN frame. Rate limit is 100 logs per 10 minutes. If you exceed the rate limit, the GO device will stop taking data from the IOX.</p>
+        <h4>Payload</h4>
+        <table>
```

Comment:
we should have tables embedded in a div with the .table-container class until we come up with the final styling so it can be applied later on